core/arch/x86_code_test.c

```diff
@@ -324,6 +324,73 @@ unit_test_get_zmm_caller_saved()
         0);
 }
 
+static void
+unit_test_get_opmask_caller_saved()
+{
+    /* While DynamoRIO's dr_opmask_t type is 8 bytes, the actual machine register is
+     * really only 8 bytes if the processor and OS support AVX512BW. Otherwise it is
+     * 2 Bytes.
+     */
+    dr_opmask_t ref_buffer[MCXT_NUM_OPMASK_SLOTS];
+    dr_opmask_t get_buffer[MCXT_NUM_OPMASK_SLOTS];
+    ASSERT(sizeof(dr_opmask_t) == OPMASK_REG_SIZE);
```

Comment:
C11 apparently added a `_Static_assert` (according to [SO](https://stackoverflow.com/questions/3385515/static-assert-in-c)), but I don't know if we want to use it.

Reply:
I think we're on gnu99 in C. The test is using "ASSERT" at the other places, I suggest to leave it for now?

Reply:
Works for me, not a big deal.